extra-files/cloud_setup/tool_sheds_conf.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
+<tool_sheds>
```

Comment:
This is OK for now, but I think we should have a small task that can write out this file. I believe we can even supply this file as .yaml nowadays, but I may be wrong.